### ISSUE TITLE
nsim argument for rshift

### ISSUE THREAD

**Issue by mhesselbarth** [spatstat/spatstat#130]
Hi, 

To simulate several patterns using the `rshift.ppp()` function, a `nsim` argument (similar to `rlabel()`) would be great. If I'm not mistaken, currently the `rshift()` function has to be looped.

```
# current solution
result_list <- vector(mode = "list", length = 199)
for(i in 1:199) {
  result_list[[i]] <- rshift.ppp(amacrine, which = "on", edge = "torus")
}

# allowing this would be nice
# result_list <- rshift.ppp(amacrine, which = "on", edge = "torus", nsim = 199)
```

Thanks,
Max

**Comment by baddstats** [comment]
This is now implemented, in `spatstat 1.64-0.001`

**Comment by mhesselbarth** [comment]
Great, thanks 👍
